README.md

```diff
@@ -1122,7 +1124,7 @@ Table of Contents
    * [FaunaDB](https://fauna.com/) — Serverless cloud database, with native GraphQL, multi-model access and daily free tiers up to 100 MB
    * [HarperDb](https://harperdb.io/) — Serverless cloud database, with dynamic schema based on JSON, 3000 IOPS with 1GB storage
    * [heroku.com](https://www.heroku.com/) — PostgreSQL as a service, up to 10,000 rows and 20 connections free (provided as an "addon," but can be attached to an otherwise empty app and accessed externally)
-   * [Upstash](https://upstash.com/) — Serverless Redis with free tier up to 10,000 requests per day, 256MB max database size, and 20 concurrent connections
+   * [Upstash](https://upstash.com/) — Serverless Redis with free tier up to 10,000 requests per day, 256MB max database size
```

Comment:
we can remove this one now - we list services once only

Reply:
That's a different product with a different free tier.

Reply:
I see `pusher.com` also added under `BaasS` and `Messaging`, so I assumed it was fine to add different products under different categories.

Reply:
We used to and are removing them as we come across them - the list is enormous already so we do not add more such multi listings.

Reply:
Neither is great lets be honest, but lets pick which one you think is best.  It's a difficult situation but with the size of the list we gotta make some choices